src/controllers/midi/hss1394controller.h

```diff
@@ -51,7 +51,8 @@ class Hss1394Controller : public MidiController {
     virtual int close();
 
   private:
-    void sendWord(unsigned int word);
+    void sendShortMsg(unsigned char status, unsigned char byte1,
+                      unsigned char byte2);
```

Comment:
The way it was before, with sendShortMsg() passing the computed word to sendWord(), the sendWord() function that actually did the sending would have to undo the operations of sendShortMsg() to print the three separate MIDI bytes.
https://github.com/mixxxdj/mixxx/pull/1004/files#diff-0212d99488d889be2b8194be8f10eed0R178